jenkins/java/ql/src/WebMethodMissingPermissionCheck.md

```diff
@@ -0,0 +1,7 @@
+A Stapler web method does not perform a permission check. This can be problem if the method returns private information, or has side effects. See [the documentation](https://www.jenkins.io/doc/developer/security/form-validation/).
+
+## False Positive Guidance
```

Comment:
```suggestion
### False Positive Guidance
```
(subjective, I found this title a little bit too big compared to the rest)

Reply:
Good question, it's unclear (and I think undocumented) which level makes semantic sense.

Reply:
For direct comparison, doesn't make much of a difference other than the line: 
<img width="1542" alt="Screenshot 2022-02-18 at 14 20 57" src="https://user-images.githubusercontent.com/1831569/154690378-637d5b19-bece-41cf-8e65-42ec27e7bec1.png">